Diffusion/Common/State.swift

```diff
@@ -126,4 +126,42 @@ class Settings {
             return ComputeUnits(rawValue: current)
         }
     }
+
+    public func applicationSupportURL() -> URL {
+        let fileManager = FileManager.default
```

Comment:
I suppose this comes from #60, doesn't it?

Reply:
I was aligning this across a few PRs to make it easier to manually merge across as it is rather central to the needs of a few PRs.